### PR TITLE
Automatically sort imports 

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Manually sorted imports 📚 

We've started doing more import sorting in PRs, this is great but it can add extra noise for reviewers so I think it's nice to agree to a sorting pattern and apply it across the repo in one fell swoop.

## Description
This PR adds some sorting config to the eslintrc.

Sorting order is up for discussion so feel free to provide suggestions.

This is the most logic sorting pattern I could find online (it's not perfect, see node imports and non-relative internal imports and bundled with external imports. But consistent is better than perfect 🙏 )

### Double check

- [x] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

File: web/.eslintrc.json [hunk 1]
```diff
@@ -20,6 +20,25 @@
...
+      "error",
+      {
+        "groups": ["external", "internal"],
+        "pathGroups": [
+          {
+            "pattern": "react",
+            "group": "external",
+            "position": "before"
+          }
+        ],
```
Comment (VIKTORVAV99) [inline web/.eslintrc.json:33]: Should we add a group for "node*" imports as well? Would be nice to group all the internal stuff seperate from our code imports.

**Comment by madsnedergaard** [comment]
Love the idea! :)
Did you look into using https://github.com/lydell/eslint-plugin-simple-import-sort instead?
Here's a list of how they differ: https://github.com/lydell/eslint-plugin-simple-import-sort#how-is-this-rule-different-from-importorder

File: web/.eslintrc.json [hunk 2]
```diff
@@ -20,6 +20,25 @@
     "prettier"
   ],
   "rules": {
+    "import/order": [
+      "error",
+      {
+        "groups": ["external", "internal"],
```
Comment (madsnedergaard) [inline web/.eslintrc.json:26]: If we stick with `import/order` (see other comment), I think we should use the full grouping they suggest:
`["builtin", "external", "internal", "parent", "sibling", "index", "object", "type"]`

**Comment by tonypls** [comment]
Closing to re-do with https://github.com/lydell/eslint-plugin-simple-import-sort

**Pull request closed without merging**